src/components/ui-shell/_shell-switcher.scss

```diff
@@ -0,0 +1,160 @@
+@import '../../globals/scss/functions';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/vars';
+@import '../../globals/scss/typography';
+@import 'theme';
+@import 'functions';
+
+@mixin shell-switcher {
+  .#{$prefix}--panel--overlay {
+    position: fixed;
+    top: units(6);
+    left: unset;
+    right: 0;
+    bottom: 0;
+    width: units(40);
+    will-change: transform;
+    transform: translate3d(100%, 0, 0);
+    padding: 1rem 0;
+    overflow-y: auto;
+    z-index: 1000;
+    background-color: $shell-header-bg-02;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
+    height: 100%;
+    overflow-x: hidden;
+  }
+
+  .#{$prefix}--panel--expanded {
+    transform: translate3d(0, 0, 0);
+    transition: transform 0.11s cubic-bezier(0.2, 0, 0.38, 0.9);
+  }
+
+  .#{$prefix}--switcher__search-container {
+    padding: 0 units(2);
+    margin-bottom: units(3);
+  }
+
+  .#{$prefix}--search--shell input {
+    background-color: #fff;
+  }
+
+  .#{$prefix}--switcher__item {
+    padding: 3px;
+  }
+
+  .#{$prefix}--switcher__subheader,
+  .#{$prefix}--switcher__all-btn {
+    @include typescale('omega');
+    padding: units(1);
+    color: $shell-header-text-03;
+  }
+
+  .#{$prefix}--switcher__subheader {
+    padding-left: units(6.9);
```

Comment:
The unit values should be whole numbers, right? Or am I missing something?

Reply:
It should yes but the subheader and link to "show all" weren't aligning with the list items and I wasn't sure how to go about it. Do you have any thoughts on this.